pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -184,15 +191,13 @@ func (s *Installer) Install(ctx context.Context, bundle releasev1alpha1.Tinkerbe
 			deploy: true,
 			image:  bundle.TinkerbellStack.Rufio.URI,
 		},
-		loadBalancer: map[string]interface{}{
-			"enabled": s.loadBalancer,
-			"ip":      tinkerbellIP,
-		},
 		kubevip: map[string]interface{}{
```

Comment:
why were loadBalancer and kubevip separate over here before?

Reply:
previously, the `loadBalancer` field was being used to manage both `kube-vip` and `envoy` deployments.
Now that we have separate flags to control them separately, the `loadBalancer` field isn't needed anymore

Reply:
tinkerbellIP should be made optional then